model/template.go

```diff
@@ -16,11 +16,11 @@ type Template struct {
 	Name           string `json:"name"`
 	IsSystem       string `json:"isSystem"`
 	Description    string `json:"description"`
-	DefaultVersion string `json:"defaultVersion" yaml:"version"`
+	DefaultVersion string `json:"defaultVersion" yaml:"default_version"`
```

Comment:
@deniseschannon Should we change this to `default_version`? Or just leave as `version`?